README.md

```diff
@@ -8,6 +8,15 @@ This demo showcases how you can perform the following actions :
 
 - Request Data
 
+
```

Comment:
Can you fix the typo/white space line 3 above:
```md
[...] following actions:
```
Instead of
```md
[...] following actions :
```

Reply:
resolved